Sources/STARSDK/Cryptography/CryptoParams.swift

```diff
@@ -0,0 +1,12 @@
+//  Created by Loïc Gardiol on 30.03.20.
+
+import Foundation
+
+struct GlobalParameters {
+    static let epoch0TimeInterval: TimeInterval = 1585475061.0
+    static let epochDuration: TimeInterval = 15.0 * 60.0
+    static let nbStoredEpochs: UInt32 = 21
```

Comment:
Maybe we should rename that. I don't see how this value is related to epochs (we have 96 epochs a day right?). Something like storedDailySecretKeys or similiar.

Reply:
Coming from the prototype code, this was for code simplicity in the epoch rotation code.
But sure, we can put a number of days and compute the resulting number of epochs in the code rotation part :-)

Reply:
We don't have to compute anything right? I think the name is just wrong :) 
This value is used to determine, if we should drop SecretKeys (we have one a day right?). So it has nothing to do with epochs (we have 96 epochs a day each of which lasts for 15 minutes), or am I confusing something?
Hence, I would suggest to use nbStoredDailySecrets instead.

Reply:
You are right. Sorry, I got stuck on the previous design, where there was 1 Secret Key per epoch. Then the code for has to changed, it still implements the previous design.